can-define.js

```diff
@@ -281,7 +283,7 @@ make = {
 				if (newVal !== current) {
 					setData.call(this, newVal);
 
-					canEvent.dispatch.call(this, {
+					canEvent.trigger.call(this, {
```

Comment:
This a problem, using .trigger makes computes not work (because domDispatch calls the handlers like DOM event handlers and canEvent.dispatch calls them like compute handler).

So I'm not sure how to fix this without checking to see if the event is for a property or not.